internal/codegen/tpl_stencil.go

```diff
@@ -182,32 +181,73 @@ func (s *TplStencil) ReadFile(name string) (string, error) {
 	return string(b), nil
 }
 
+type ReadDirEntry struct {
+	Name  string
+	IsDir bool
+}
```

Comment:
**thought(non-blocking):** Is it worth just returning the [`os.FileInfo`](https://pkg.go.dev/os#FileInfo) interface instead?

Reply:
can't -- that structure uses methods

Reply:
I'm pretty sure `file.Name` just gets called automatically in a Go template, or do you mean something else?